plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -310,19 +311,24 @@ private static String buildSplitPath(String tableLocation, AddFileEntry addActio
     {
         // paths are relative to the table location and are RFC 2396 URIs
         // https://github.com/delta-io/delta/blob/master/PROTOCOL.md#add-file-and-remove-file
-        URI uri = URI.create(addAction.getPath());
-        String path = uri.getPath();
+        String path = addAction.getPath();
+
+        // checking if path is absolute
+        boolean isAbsolutePath = URI.create(path).isAbsolute();
+
+        // if not absolute, appending to tableLocation
+        if (!isAbsolutePath) {
+            path = Locations.appendPath(tableLocation, path);
```

Comment:
nit: These code comments aren't helpul to me because it's obvious. I would remove them.

Reply:
@ebyhr removed the comments wherein it seemed obvious